contracts/FungibleToken-v2-ContractInterface.cdc

```diff
@@ -0,0 +1,14 @@
+import FungibleToken from "./FungibleToken-v2.cdc"
+
+pub contract interface FungibleTokenInterface {
+
+    /// Contains the total supply of the fungible token
+    pub var totalSupply: UFix64
```

Comment:
Should this be a single UFix64 if we can have multiple types of vaults? Won't that get them mixed up?

Reply:
updated it in a recent commit to be a dictionary